src/pip/_internal/pep425tags.py

```diff
@@ -276,8 +294,16 @@ def get_supported(versions=None, noarch=False, platform=None,
             else:
                 # arch pattern didn't match (?!)
                 arches = [arch]
-        elif platform is None and is_manylinux1_compatible():
-            arches = [arch.replace('linux', 'manylinux1'), arch]
+        elif arch_prefix == 'manylinux2010':
+            # manylinux1 wheels run on manylinux2010 systems.
```

Comment:
As per the discussion on the previous PR, we should add a link to the relevant PEP section in this comment, and qualify it as "Most" wheels being compatible.

Reply:
I've updated the comment here. Anything else you'd like to see added? Do you have a preferred method for including log urls?